packages/phoenix-evals/src/phoenix/evals/models/openai.py

```diff
@@ -279,27 +280,43 @@ def _init_rate_limiter(self) -> None:
         )
 
     def _build_messages(
-        self, prompt: MultimodalPrompt, system_instruction: Optional[str] = None
+        self, prompt: MultimodalPrompt, data_fetcher: Optional[Callable[[str], Audio]] = None, system_instruction: Optional[str] = None
```

Comment:
Also note that the type annotation on the callable does not match the docstring, it's best if users return a familiar type instead of an internal one I think